Cannot Register_Serializer from outside of inet::serializers namespace

When trying to register new serializers for custom packets I experienced problems when calling Register_Serializer:

```
In file included from /opt/omnetpp/omnetpp-4.6/include/globals.h:19:0,
                 from /opt/omnetpp/omnetpp-4.6/include/cobjectfactory.h:20,
                 from /opt/omnetpp/omnetpp-4.6/include/omnetpp.h:23,
                 from ../../inet/src/inet/common/INETDefs.h:25,
                 from ../../inet/src/inet/common/serializer/SerializerBase.h:22,
                 from common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.h:21,
                 from common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.cc:18:
common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.cc: In function 'void CoRE4INET::serializer::{anonymous}::__onstartup_func_36()':
common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.cc:35:1: error: 'serializers' was not declared in this scope
 Register_Serializer(EthernetIIFrameWithQTag, inet::serializer::LINKTYPE, inet::serializer::LINKTYPE_ETHERNET,
 ^
common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.cc:35:1: note: suggested alternative:
In file included from common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.h:21:0,
                 from common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.cc:18:
../../inet/src/inet/common/serializer/SerializerBase.h:141:44: note:   'inet::serializer::serializers'
 INET_API extern SerializerRegistrationList serializers; ///< List of packet serializers (SerializerBase)
                                            ^
Makefile:382: recipe for target '../out/gcc-debug/src/common/serializer/ethernet/CoRE4INET_EtherFrameWithQTagSerializer.o' failed
```
The problem is that the define is not namespace neutral:
`
#define Register_Serializer(SERIALIZABLECLASSNAME, PROTOCOLGROUP, PROTOCOLID, SERIALIZERCLASSNAME)   \
        EXECUTE_ON_STARTUP(serializers.add(opp_typename(typeid(SERIALIZABLECLASSNAME)), \
                PROTOCOLGROUP, PROTOCOLID, new SERIALIZERCLASSNAME(#SERIALIZABLECLASSNAME)););
`

should be:

`
#define Register_Serializer(SERIALIZABLECLASSNAME, PROTOCOLGROUP, PROTOCOLID, SERIALIZERCLASSNAME)   \
        EXECUTE_ON_STARTUP(inet::serializer::serializers.add(opp_typename(typeid(SERIALIZABLECLASSNAME)), \
                PROTOCOLGROUP, PROTOCOLID, new SERIALIZERCLASSNAME(#SERIALIZABLECLASSNAME)););
`

--- a/src/inet/common/serializer/SerializerBase.h
+++ b/src/inet/common/serializer/SerializerBase.h
@@ -141,7 +141,7 @@ class INET_API SerializerRegistrationList : public cNamedObject, noncopyable
 INET_API extern SerializerRegistrationList serializers; ///< List of packet serializers (SerializerBase)
 
 #define Register_Serializer(SERIALIZABLECLASSNAME, PROTOCOLGROUP, PROTOCOLID, SERIALIZERCLASSNAME)   \
-        EXECUTE_ON_STARTUP(serializers.add(opp_typename(typeid(SERIALIZABLECLASSNAME)), \
+        EXECUTE_ON_STARTUP(inet::serializer::serializers.add(opp_typename(typeid(SERIALIZABLECLASSNAME)), \
                 PROTOCOLGROUP, PROTOCOLID, new SERIALIZERCLASSNAME(#SERIALIZABLECLASSNAME)););
 
 } // namespace serializer

